contracts/mocks/docs/account/paymaster/MyPaymasterECDSA.sol

```diff
@@ -0,0 +1,14 @@
+// contracts/MyPaymasterECDSA.sol
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {PaymasterSigner, EIP712} from "../../../../account/paymaster/PaymasterSigner.sol";
+import {PackedUserOperation} from "@openzeppelin/contracts/interfaces/draft-IERC4337.sol";
+import {SignerECDSA} from "../../../../utils/cryptography/SignerECDSA.sol";
+
+contract MyPaymasterECDSA is PaymasterSigner, SignerECDSA {
+    constructor(address paymasterSignerAddr) EIP712("MyAccountECDSA", "1") {
+        _setSigner(paymasterSignerAddr);
+    }
+}
```

Comment:
```suggestion

    // Staking & unstacking logic (permissioned)
}
```

Reply:
he _authorizeWithdraw should make it clearer, do you agree?